manifests/06_role_binding.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:openshift:operator:cert-manager
```

Comment:
This is something I was scratching my head about...

In the Cluster Authentication Operator, we use `system:openshift:operator:authentication` (see [here](https://github.com/openshift/cluster-authentication-operator/blob/master/manifests/04_roles.yaml)). As you can see, there's no `operator` suffix there. Probably, it's because it's somewhere in the middle of the Cluster Role Binding name. 

So I applied a similar convention here. Could you please clarify, whether it's fine or not? If not - why?

Reply:
oh, I see, I think I missed the middle `:operator:` originally. Keep as is, then :+1: